src/test/java/elizCurtnazar/tests/tema7/MakeAppointmentScenariosTest.java

```diff
@@ -12,7 +12,7 @@
 public class MakeAppointmentScenariosTest extends BaseTest{
 
     @Test
-    public void clickMakeAppointmentButton() {
+    public void clickMakeAppointmentButton(String expectedResult, String actualResults) {
```

Comment:
You are missing the negative scenario for appointment -> 
![image](https://user-images.githubusercontent.com/18215854/214047586-9f56b814-2e10-425f-bb33-0f23b4f1f6c5.png)